### DISCUSSION TITLE
Make the cursor always a few lines aways from the toppest or the bottomest line?

### DISCUSSION THREAD

**Discussion by 2533245542** [JetBrains/ideavim#427] [Q&A]
In a vim I used before, when I pressed H the screen scrolls up, but the cursor was always a few lines away from the toppest line.

When I did ctrl-u. the cursor was not at the top but always stayed a few lines below.

Is there a way to do this in idea vim?

I am not sure if I described this right.

**Comment by AlexPl292** [comment]
Hi, I suppose your vim config had a `scrolloff` option. IdeaVim supports this option too. You can add `set scrolloff=5` to keep 5 lines above the caret.
